hudi-common/src/main/java/org/apache/hudi/timeline/TimelineServiceClient.java

```diff
@@ -61,7 +61,7 @@ protected Response executeRequest(Request request) throws IOException {
     String url = builder.toString();
     LOG.debug("Sending request : ({})", url);
     org.apache.http.client.fluent.Response response = get(request.getMethod(), url, timeoutMs);
-    return new Response(response.returnContent().asString());
+    return new Response(response.returnContent().asStream());
```

Comment:
The jackson docs mention avoiding intermediate string representation is just unnecessary translation of the data so switching to input stream for performance.